### DISCUSSION TITLE
Please considering adding the ability to bulk-add issues to projects, similar to bulk-add for tags

### DISCUSSION THREAD

**Discussion by joe-brilliant** [community/community#13713] [Projects and Issues]
Please consider adding the ability to bulk-add issues to projects, similar to bulk-add for tags

**Comment by schalkneethling** [comment]
This is now in beta - https://github.blog/changelog/2022-04-07-the-new-github-issues-april-7th-update/

**Reply by fredguth** [reply]
If you added the items in the project as "drafts", can you bulk insert in a project?
